x/emissions/module/rewards/topic_rewards.go

```diff
@@ -188,7 +141,8 @@ func PickChurnableActiveTopics(
 			}
 		}
 		// Check Reputer Close Cadence
-		if k.CheckReputerCloseCadence(block, topic) {
+		//if k.CheckReputerCloseCadence(block, topic)
```

Comment:
This I think we still want to keep because there is a `ground_truth_lag` period of time in which there are worker nonces but no reputer nonces.

Reply:
You're right for reproducing the previous behaviour @kpeluso 
However opened a convo ab this separately on the desired behaviour here, so we can clearly reproduce it in docs with 100% certainty on intent.